lighthouse-core/gather/gatherers/console-log-entry.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Gathers console deprecation and intervention warnings logged by Chrome.
+ */
+
+'use strict';
+
+const Gatherer = require('./gatherer');
+
+class ConsoleLogEntry extends Gatherer {
```

Comment:
Not a great name, is it? Hmm. We need something in between console messages and console logs entries only produced by chrome. `LogEntries` is better but it's still ambiguous. What about 'ChromeConsoleLogs`?